installers/npm/binary.js

```diff
@@ -176,7 +179,7 @@ class Binary {
         });
       })
       .then(() => {
-        fs.renameSync(join(this.installDirectory, name), this.binaryPath);
+        fs.renameSync(join(this.installDirectory, this.raw_name), this.binaryPath);
```

Comment:
This is the actual bug-fix here, make sure we use the raw_name when renaming (so `rover.exe` and not `rover`)